relay-config/src/config.rs

```diff
@@ -87,6 +116,34 @@ pub enum ConfigErrorKind {
     /// compiled without the processing feature.
     #[fail(display = "was not compiled with processing, cannot enable processing")]
     ProcessingNotAvailable,
+    /// An override from a environment variable or a command line arg could not be
+    /// parsed into the proper type
+    #[fail(display = "Invalid override")]
+    InvalidOverride,
+}
+
+/// Structure used to hold information about configuration overrides via
+/// CLI parameters or environment variables
+#[derive(Debug, Default)]
+pub struct OverridableConfig {
+    /// the upstream for the Relay
```

Comment:
_nit_: Can you use the same doc strings here as on the original fields?

Reply:
ok